utils/getClaimStatus.tsx

```diff
@@ -7,66 +7,115 @@ import { ClaimStatusContent, I18nString, TextOptionalLink, TransLineProps } from
 import { ScenarioType } from './getScenarioContent'
 import getUrl, { UrlType } from './getUrl'
 
+type StepType = 'your-next-steps' | 'edd-next-steps'
+
+interface ClaimStatusScenarioJson {
+  heading: string
+  summary: TextOptionalLink
+  'your-next-steps': TextOptionalLink[]
+  'edd-next-steps': TextOptionalLink[]
+}
+
+/**
+ * Convert a ScenarioType into the string key used in json translation files.
+ */
+export function scenarioToString(scenarioType: ScenarioType): string {
+  return ScenarioType[scenarioType].toLowerCase()
+}
+
 /**
  * Helper function to get shared Claim Status translation string prefix.
  */
-function getTranslationPrefix(scenarioType: ScenarioType): I18nString {
-  return `claim-status:scenarios.${ScenarioType[scenarioType].toLowerCase()}`
+function buildTranslationPrefix(scenarioType: ScenarioType): I18nString {
+  return `claim-status:scenarios.${scenarioToString(scenarioType)}`
 }
 
 /**
  * Get Claim Status heading.
  */
-export function getClaimStatusHeading(scenarioType: ScenarioType): I18nString {
-  return getTranslationPrefix(scenarioType) + '.heading'
+export function buildClaimStatusHeading(scenarioType: ScenarioType): I18nString {
+  return buildTranslationPrefix(scenarioType) + '.heading'
 }
 
 /**
- * Get Claim Status summary.
- *
- * Note: This function has a lot of verbosity due to Typescript + eslint.
+ * Build a list of urls from keys in the json translation files.
  */
-export function getClaimStatusSummary(scenarioType: ScenarioType): TransLineProps {
-  const transLineProps: TransLineProps = {
-    i18nKey: getTranslationPrefix(scenarioType) + '.summary.text',
+export function buildTransLineLinks(linkKeys: string[] | undefined): string[] {
```

Comment:
I wanted to allow `linkKeys` to be `undefined` so that we don't have to populate the translation files with a bunch of lines that look like this:

```
"your-next-steps": [
  {
    "text": "Text here",
    "links": [] <--- noisy empty arrays
  },
  ...
],
```